### PR TITLE
RDoc-1888-Grafana

### PR DESCRIPTION
**Pull request by morhilai**

### CODE REVIEW

File: Documentation/5.2/Raven.Documentation.Pages/server/administration/monitoring.markdown [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+{PANEL: }
+
+[Telegraf](https://www.influxdata.com/time-series-platform/telegraf/) is a popular data collection 
+and processing agent designed to work with time series data. Version 1.18 of Telegraf has a new 
+plugin for RavenDB that collects data from RavenDB's monitoring endpoints. The recommended use 
+for the RavenDB plugin is to have Telegraf output to [InfluxDB](https://www.influxdata.com/products/influxdb/), 
+and from there the data can be queried by [Grafana](https://grafana.com/) and displayed on your own 
+data tracking dashboard. But this feature is flexible - Telegraf can output data to other destinations. 
+
+The monitoring endpoints output data in JSON format. There are three endpoints:
```
Comment (ml054) [inline Documentation/5.2/Raven.Documentation.Pages/server/administration/monitoring.markdown:26]: 4 endpoints (collections is missing)

File: Documentation/5.2/Raven.Documentation.Pages/server/administration/monitoring.markdown [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+for the RavenDB plugin is to have Telegraf output to [InfluxDB](https://www.influxdata.com/products/influxdb/), 
+and from there the data can be queried by [Grafana](https://grafana.com/) and displayed on your own 
+data tracking dashboard. But this feature is flexible - Telegraf can output data to other destinations. 
+
+The monitoring endpoints output data in JSON format. There are three endpoints: 
+* `<your server URL>/admin/monitoring/v1/server`
+* `<your server URL>/admin/monitoring/v1/databases`
+* `<your server URL>/admin/monitoring/v1/indexes`
+
+The root OID for these endpoints is: `1.3.6.1.4.1.45751.1.1`.
```
Comment (ml054) [inline Documentation/5.2/Raven.Documentation.Pages/server/administration/monitoring.markdown:31]: remove - it isn't snmp

File: Documentation/5.2/Raven.Documentation.Pages/server/administration/monitoring.markdown [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+The monitoring endpoints output data in JSON format. There are three endpoints: 
+* `<your server URL>/admin/monitoring/v1/server`
+* `<your server URL>/admin/monitoring/v1/databases`
+* `<your server URL>/admin/monitoring/v1/indexes`
+
+The root OID for these endpoints is: `1.3.6.1.4.1.45751.1.1`.  
+
+The following is a list of JSON fields returned by the endpoints:  
+
+| Endpoint suffix | OID suffix | Field Name | Description |
```
Comment (ml054) [inline Documentation/5.2/Raven.Documentation.Pages/server/administration/monitoring.markdown:35]: remove OID

File: Documentation/5.2/Raven.Documentation.Pages/server/administration/monitoring.markdown [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+| `server` | `1.4` | `ServerProcessId` | Server process ID |
+| `server` | `1.5.1` | `ProcessUsage` | Process CPU usage in % |
+| `server` | `1.5.2` | `MachineUsage` | Machine CPU usage in % |
+| `server` | `1.5.3.1` | `` | ~CPU Credits Base~ |
+| `server` | `1.5.3.2` | `` | ~CPU Credits Max~ |
+| `server` | `1.5.3.3` | `` | ~CPU Credits Remaining~ |
+| `server` | `1.5.3.4` | `` | ~CPU Credits Gained Per Second~ |
+| `server` | `1.5.3.5` | `` | ~CPU Credits Background Tasks Alert Raised~ |
+| `server` | `1.5.3.6` | `` | ~CPU Credits Failover Alert Raised~ |
+| `server` | `1.5.3.7` | `` | ~CPU Credits Any Alert Raised~ |
```
Comment (ml054) [inline Documentation/5.2/Raven.Documentation.Pages/server/administration/monitoring.markdown:53]: it isn''t available remove

File: Documentation/5.2/Raven.Documentation.Pages/server/administration/monitoring.markdown [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+| `server` | `1.14.2` | `ThreadPoolAvailableCompletionPortThreads` | Number of available completion port threads in the thread pool |
+| `server` | `1.15.1` | `TcpActiveConnections` | Number of active TCP connections |
+| `server` | `3.1.1` | `NodeTag` | Current node tag |
+| `server` | `3.1.2` | `NodeState` | Current node state |
+| `server` | `3.2.1` | `CurrentTerm` | Cluster term |
+| `server` | `3.2.2` | `Index` | Cluster index |
+| `server` | `3.2.3` | `Id` | Cluster ID |
+| `server` | `5.1.1` | `TotalCount` | Number of all databases |
+| `server` | `5.1.2` | `LoadedCount` | Number of loaded databases |
+| `server` | `5.1.3` | `` | ~Time since oldest backup~ |
```
Comment (ml054) [inline Documentation/5.2/Raven.Documentation.Pages/server/administration/monitoring.markdown:94]: remove?

Comment (ml054) [reply Documentation/5.2/Raven.Documentation.Pages/server/administration/monitoring.markdown:94]: in general please go thought plugin code to see what we collect

Comment (ml054) [reply Documentation/5.2/Raven.Documentation.Pages/server/administration/monitoring.markdown:94]: spreadsheet can be useful for description lookup

File: Documentation/5.2/Raven.Documentation.Pages/server/administration/monitoring.markdown (hunk 1 above)
Comment (morhilai) [reply Documentation/5.2/Raven.Documentation.Pages/server/administration/monitoring.markdown:26]: fixed

File: Documentation/5.2/Raven.Documentation.Pages/server/administration/monitoring.markdown (hunk 2 above)
Comment (morhilai) [reply Documentation/5.2/Raven.Documentation.Pages/server/administration/monitoring.markdown:31]: fixed

File: Documentation/5.2/Raven.Documentation.Pages/server/administration/monitoring.markdown (hunk 3 above)
Comment (morhilai) [reply Documentation/5.2/Raven.Documentation.Pages/server/administration/monitoring.markdown:35]: fixed

File: Documentation/5.2/Raven.Documentation.Pages/server/administration/monitoring.markdown (hunk 4 above)
Comment (morhilai) [reply Documentation/5.2/Raven.Documentation.Pages/server/administration/monitoring.markdown:53]: fixed

File: Documentation/5.2/Raven.Documentation.Pages/server/administration/monitoring.markdown (hunk 5 above)
Comment (morhilai) [reply Documentation/5.2/Raven.Documentation.Pages/server/administration/monitoring.markdown:94]: Will do. Fixed

File: Documentation/5.2/Raven.Documentation.Pages/server/administration/monitoring.markdown [hunk 6]
```diff
@@ -23,125 +23,115 @@ for the RavenDB plugin is to have Telegraf output to [InfluxDB](https://www.infl
...
-| `indexes` | `5.2.X4.Y.13` | `Status` | Index status |
-| `indexes` | `5.2.X4.Y.14` | `MappedPerSec` | Number of maps per second (one minute rate) |
-| `indexes` | `5.2.X4.Y.15` | `ReducedPerSec` | Number of reduces per second (one minute rate) |
-| `indexes` | `5.2.X4.Y.16` | `Type` | Index type |
-| `indexes` | N/A | `Lagtime` | Indexing Lag Time |
-| `indexes` | `` | `EntriesCount` | Number of entries in the index |
-| `databases` | `` | `TimeSinceLastBackupInSec` | LastBackup |
+| Endpoint suffix | Field Name | Description |
+| - | - | - |
+| `server` | `ServerUrls` | Server URL |
```
Comment (ml054) [inline Documentation/5.2/Raven.Documentation.Pages/server/administration/monitoring.markdown:36]: Maybe we should include field name which we have in telegraf, see: https://github.com/influxdata/telegraf/blob/master/plugins/inputs/ravendb/ravendb.go#L188

Comment (morhilai) [reply Documentation/5.2/Raven.Documentation.Pages/server/administration/monitoring.markdown]: Done

File: Documentation/5.2/Raven.Documentation.Pages/server/administration/monitoring.markdown [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+| `indexes` | `time_since_last_indexing_in_sec` | Time since last indexing |
+| `indexes` | `lock_mode` | Index lock mode |
+| `indexes` | `is_invalid` | Indicates if index is invalid |
+| `indexes` | `status` | Index status |
+| `indexes` | `mapped_per_sec` | Number of maps per second (one minute rate) |
+| `indexes` | `reduced_per_sec` | Number of reduces per second (one minute rate) |
+| `indexes` | `type` | Index type |
+| `indexes` | `lagtime` | Indexing Lag Time |
+| `indexes` | `entries_count` | Number of entries in the index |
+| `databases` | `time_since_last_backup_in_sec` | LastBackup |
```
Comment (ml054) [inline Documentation/5.2/Raven.Documentation.Pages/server/administration/monitoring.markdown:129]: let's sort those by first column + looks like data from collections endpoint is missing.

Comment (morhilai) [reply Documentation/5.2/Raven.Documentation.Pages/server/administration/monitoring.markdown]: Done

**Review by ml054 — APPROVED** [review]

**Pull request merged**